packages/xstate-fsm/rollup.config.js

```diff
@@ -1,8 +1,83 @@
 import typescript from 'rollup-plugin-typescript2';
+import babel from '@rollup/plugin-babel';
 import { terser } from 'rollup-plugin-terser';
 import rollupReplace from 'rollup-plugin-replace';
 import fileSize from 'rollup-plugin-filesize';
 
```

Comment:
These changes based on the hypothesis that the package build process should be similar in case of  `core` and `xstate-fsm`. I did not make any fine-grained understanding of the build.

It seems solving the SvelteKit issue https://github.com/statelyai/xstate/issues/2458, and does not have regression in case of React and Vue.